### DISCUSSION TITLE
有关弹幕接口的问题

### DISCUSSION THREAD

**Discussion by Lp17863564** [Nemo2011/bilibili-api#93] [Q&A] [answered]
单纯使用示例获取弹幕结果：
for d in danmakus:
UnboundLocalError: local variable 'danmakus' referenced before assignment

**Comment by Nemo2011** [comment]
请展示一下你的源代码（全部）

**Reply by Lp17863564** [reply]
#[示例：下载视频的弹幕]
from bilibili_api import ass, sync
sync(ass.make_ass_file_danmakus_protobuf("BV1AV411x7Gs", 0, "out.ass"))
感谢回复

**Reply by Nemo2011** [reply, marked as answer]
sync(ass.make_ass_file_danmakus_protobuf(video.Video("BV1AV411x7Gs"), 0, "out.ass"))

文档忘记更新了

**Reply by Lp17863564** [reply]
收到，谢谢！
